tachyon/math/base/big_int.h

```diff
@@ -33,6 +33,8 @@ constexpr size_t LimbsAlignment(size_t x) {
 
 }  // namespace internal
 
+// BigInt is a fixed size array of uint64_t, capable of holding up to |N| limbs,
```

Comment:
How about |BigInt|? If you agree, can you fix all the other comments that needs fix as well?

Reply:
Maybe just BigInt is okay since it's capitalized like the naming convention in cpp. I might have to change the comments in #71._